pilot/pkg/networking/v1alpha3/listener.go

```diff
@@ -480,26 +486,37 @@ func buildHTTPListener(opts buildHTTPListenerOpts) *xdsapi.Listener {
 				},
 			},
 		}
-	} else {
-		rds = opts.rdsConfig
-	}
+		connectionManager = &http_conn.HttpConnectionManager{
+			CodecType: http_conn.AUTO,
+			AccessLog: []*accesslog.AccessLog{
```

Comment:
remove this since it is set below
or create the access log above and set it here